### ISSUE TITLE
Cursor click indication

### ISSUE THREAD

**Issue by CoryDuncan** [aaron-elkins/capture-gif-issue#6]
It would be nice if the app showed some indication of a cursor click. Maybe hiding the green circle or removing the green circle fill (leaving an outline), when a click happens. Otherwise for animated screen captures, it can sometimes be difficult to tell where/when a click has occurred.

**Comment by orklann** [comment]
Hi @CoryDuncan 

Sorry for late reply, your idea is nice, and I tried your idea before but failed.

Because GIF animation is generally in low FPS or it will be laggy, and we saw some tracking of mouse clicking in some screen recording app that export to video format, and those recording and formats are generally in hight FPS. 

If the result GIF is in low FPS, then we can not always capture the mouse clicking event in every frame (That's to say, when you click your mouse and at the same time this frame may not be captured into the GIF), so in the end, the GIF file will sometimes miss a clicking event.

So sorry, this is GIF, and not video format.
